src/phoenix/trace/exporter.py

```diff
@@ -30,6 +32,52 @@ def export(self, _: Any) -> None:
         pass
 
 
+class _OpenInferenceExporter:
+    def __init__(
+        self,
+        endpoint: Optional[str] = None,
+        host: Optional[str] = None,
+        port: Optional[int] = None,
+    ) -> None:
+        """
+        A compatibility class used to configure the Phoenix endpoint for instrumentation.
+
+        Parameters
+        ----------
+        endpoint: Optional[str]
+            The endpoint of the Phoenix server (collector). This should be set if the Phoenix
+            server is running on a remote instance. It can also be set using environment
+            variable `PHOENIX_COLLECTOR_ENDPOINT`, otherwise it defaults to `http://127.0.0.1:6006`
+            Note, this parameter supersedes `host` and `port`.
+        host: Optional[str]
+            The host of the Phoenix server. It can also be set using environment
+            variable `PHOENIX_HOST`, otherwise it defaults to `127.0.0.1`.
+        port: Optional[int]
+            The port of the Phoenix server. It can also be set using environment
+            variable `PHOENIX_PORT`, otherwise it defaults to `6006`.
+        """
+        self._host = host or get_env_host()
```

Comment:
need to fix this for windows
```suggestion
        host = host or get_env_host()
        if host == "0.0.0.0":
            host = "127.0.0.1"
        self._host = host
```